internal/jobs/queue/dejq/dejq_queue.go

```diff
@@ -30,7 +30,7 @@ func init() {
 func RegisterJobs(logger *zerolog.Logger) {
 	logger.Debug().Msg("Registering job queue handlers")
 	dejqQueue.RegisterHandler(queue.TypeNoop, jobs.HandleNoop)
-	dejqQueue.RegisterHandler(queue.TypePubkeyUploadAws, jobs.HandlePubkeyUploadAWS)
+	dejqQueue.RegisterHandler(queue.TypePubkeyUploadAws, jobs.HandleEnsurePubkeyOnAWS)
```

Comment:
Can you also change the queue name? We do not persist queues yet so you can do this cheaply.

Reply:
Done